### PR TITLE
fix(loading icon): messageLoading was never being set to false

### PR DESCRIPTION
**Pull request by josephmcg**
maybe it was accidentally deleted

**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨
Permanent message loading icon
Fixes #

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by stavares843** [comment]
seems it was removed here
https://github.com/Satellite-im/Core-PWA/commit/801d9a0673d4983d11f0aa12afa7694de4de2063

cc @Jekrimo in case you wanna review

File: store/textile/actions.ts [hunk 1]
```diff
@@ -223,15 +223,17 @@ export default {
       sender: MessageRouteEnum.OUTBOUND,
       message: result,
     })
-  },/**
+    commit('setMessageLoading', { loading: false })
```
Comment (josephmcg) [inline store/textile/actions.ts:226]: mostly prettier formatting, this is my only addition

Comment (stavares843) [reply store/textile/actions.ts:226]: 🎉

**Comment by stavares843** [comment]
Captura de ecrã 2021-12-16, às 03 06 55

is about that? i thought was on purpose like animation lol 😂

**Review by stavares843 — APPROVED** [review]

**Comment by josephmcg** [comment]
exactly lol

**Review by RetroPronghorn — APPROVED** [review]

**Pull request merged**